lib/ProofSet.js

```diff
@@ -266,7 +267,7 @@ module.exports = class ProofSet {
       }
       return {verified, results};
     } catch(error) {
-      return {verified: false, error};
+      return {verified: false, error: serializeError(error)};
```

Comment:
I don't know why we're serializing errors like this -- it seems like an anti-pattern. It makes sense if we're, for example, returning the results via an HTTP API, but why lose native `Error` support?

Reply:
Maybe we don't want this behavior all the time, but having to relay these errors to HTTP is common enough that having to serialize the error with it's requisite deps in all the places is unattractive.  How do you feel about a `serializeError` boolean param to the verify method?

Reply:
Adding a special flag that will do it for you (and it's off by default) sounds good. Let's do that.

Reply:
sounds good.